### ISSUE TITLE
powerline-moe-theme line number hidden

### ISSUE THREAD

**Issue by odrevet** [kuanyui/moe-theme.el#76]
when there are to much minor mode, the line number is pushed to the right, outside the screen and is not diplayed. 
the minor mode section should be wrapped or the line number should be displayed on the left side

**Comment by choma** [comment]
I don't think this issue should be fixed here. It has nothing to do with the theme, and there are other ways to fix the problem. I personally use [diminish](https://www.emacswiki.org/emacs/DiminishedModes)
